app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -44,7 +44,7 @@ class EventsFragment : Fragment() {
                 val bundle = Bundle()
                 bundle.putLong(fragment.EVENT_ID, eventID)
                 fragment.arguments = bundle
-                activity?.supportFragmentManager?.beginTransaction()?.add(R.id.frameContainer, fragment)?.addToBackStack(null)?.commit()
+                activity?.supportFragmentManager?.beginTransaction()?.replace(R.id.frameContainer, fragment)?.addToBackStack(null)?.commit()
```

Comment:
I had to change this as there was a part in fragment_event where part of previous EventsFragment was visible making it look quite ugly.